takwimu/models/dashboard.py

```diff
@@ -30,6 +32,11 @@
 
 logger = logging.getLogger(__name__)
 
+json_data = open('takwimu/fixtures/sdg.json')
```

Comment:
Since we're now using `sdg.json`, we should replace and remove the use of `sdg.py`